addons/web/static/src/js/services/data_manager.js

```diff
@@ -406,8 +406,12 @@ return core.Class.extend({
                 if (fieldsInfo[node.attrs.name].fieldDependencies) {
                     var deps = fieldsInfo[node.attrs.name].fieldDependencies;
                     for (var dependency_name in deps) {
+                        var dependency_dict = {'name': dependency_name, 'type': deps[dependency_name].type, 'options':  deps[dependency_name].options || {}};
                         if (!(dependency_name in fieldsInfo)) {
-                            fieldsInfo[dependency_name] = {'name': dependency_name, 'type': deps[dependency_name].type, 'options':  deps[dependency_name].options || {}};
+                            fieldsInfo[dependency_name] = dependency_dict;
+                        }
+                        if (!(dependency_name in fields)) {
+                            fields[dependency_name] = dependency_dict;
```

Comment:
not sure that options should be in fields, it is probably only necessary in fieldsInfo

Reply:
The other fields (in the fields object, not fieldsInfo) don't have a 'options' key, so i believe that my above comment is relevant. I'd like to understand why it crashes without it.